dashboard/src/components/AppView/AppView.tsx

```diff
@@ -289,6 +308,23 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
   private deleteApp = (purge: boolean) => {
     return this.props.deleteApp(this.props.releaseName, this.props.namespace, purge);
   };
+
+  private getLatest() {
+    const { app } = this.props;
+    if (app.chart && app.chart.metadata && app.chart.metadata.version) {
+      let latest = app.chart.metadata.version;
+      if (this.props.latest) {
+        this.props.latest.forEach(l => {
+          // semver.compare returns -1 if v2 is bigger than v1
+          if (semver.compare(latest, l.relationships.latestChartVersion.data.version) < 0) {
+            latest = l.relationships.latestChartVersion.data.version;
+          }
+        });
+      }
+      return latest;
```

Comment:
why don't we just calculate this result in the action or reducer and pass it directly to AppView? I don't think we need to calculate it here and we don't use all the info from the latest call.